ansible_collections/arista/avd/molecule/eos_designs_unit_tests/inventory/group_vars/BGP_PEER_GROUPS_STRUCTURED_CONFIG.yml

```diff
@@ -0,0 +1,69 @@
+# Minimum config to only test the specific feature.
+# This is in no way a "good" or working config.
+# Everything is tweaked to trigger config generation of all the different peer-groups
+
+bgp_peer_groups:
+  ipv4_underlay_peers:
+    # Custom structured config added under router_bgp.peer_groups.<name> for eos_cli_config_gen
+    structured_config:
+      description: Description for ipv4_underlay_peers via structured_config
+  mlag_ipv4_underlay_peer:
+    # Custom structured config added under router_bgp.peer_groups.<name> for eos_cli_config_gen
+    structured_config:
+      description: Description for mlag_ipv4_underlay_peer via structured_config
+  evpn_overlay_peers:
+    # Custom structured config added under router_bgp.peer_groups.<name> for eos_cli_config_gen
+    structured_config:
+      description: Description for evpn_overlay_peers via structured_config
+  evpn_overlay_core:
+    # Custom structured config added under router_bgp.peer_groups.<name> for eos_cli_config_gen
+    structured_config:
+      description: Description for evpn_overlay_core via structured_config
```

Comment:
This is because the example uses ibgp and multidomain evpn stuff is not included in the ibgp templates. Conversely the mpls related peer groups aren't in the ebgp templates, so to fully cover the functionality i believe we need an example with ebgp as well, or we call it good enough.

Reply:
I guess we can call it good enough as we would need to engineer yet another convoluted set of examples just to see a custom string being printed :)

Reply:
Would appreciate  if we could let this one slip for now :)